kernel/config.txt

```diff
@@ -21,12 +21,144 @@ necessary. Information about pre-built hardware as a whole will not be covered.
+- Getting a config                                                         [1.0]
+- Modifying a config                                                       [2.0]
+- Busybox compatibility                                                    [3.0]
+- Never lose your .config ever again                                       [4.0]
+- Removing the perl requirement                                            [5.0]
+- Making an initramfs                                                      [6.0]
 
 
-[1.0] Busybox compatibility
+[1.0] Getting a config
+________________________________________________________________________________
+
+The first step in this process is creating a .config file in the Linux source
+tree. It is essentially just a text file containing KEY=value pairs (along with
+comments), each of which control something in the linux build process. Some of
+them are features to be included, some values to be hardcoded.
+
+An important subset of them are the options controlling "drivers", or sections
+of code that give the kernel the capability to interact with various filesystem,
+protocols and hardware components. Drivers configs generally have three possible
+answers: "n" to not compile the driver, "y" to compile it into the kernel
+binary, and "m" to compile them as modules (.ko files stored in
+/usr/lib/modules), that can be loaded and unloaded dynamically by the kernel as
+needed.
+
+To actully get on with the topic of creating a config, the simplest method is
+to go to the Linux source tree and run:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make config                                                              |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+This will launch an interactive shell prompt that will ask you to choose
+between several choices for all the thousands of configurations the kernel has
+(weekend goals).
+
+Another, perhaps less torturous method is just running:
+
++------------------------------------------------------------------------------+
+|                                                                              |
+|   $ make defconfig                                                           |
+|                                                                              |
++------------------------------------------------------------------------------+
+
+which will land you with a premade, fairly generic and compatible configuration
+for your architecture.
+
+Another option is to get the configuration that came with your host distribution
+(stored in /boot or exposed in /proc) and copy it to .config. It is very
+generic, obviously, and will take a lot of time to build unless significantly
+trimmed down.
```

Comment:
This can be omitted as well. I don't recommend doing this as things will likely not work at all.